### PR TITLE
chore: interactive cli improvements suggestions

### PR DESCRIPTION
**Pull request by mtsitrin**
# PR Standards

## Opening a pull request should be able to meet the following requirements

---

For Author:

- [ ]  Targeted PR against correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to Github issue with discussion and accepted design
- [ ]  Targets only one github issue
- [ ]  Wrote unit and integration tests
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)

---

For Reviewer:

- [ ]  confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  confirmed all author checklist items have been addressed

---

After reviewer approval:

- [ ]  In case targets main branch, PR should be squashed and merged.
- [ ]  In case PR targets a release branch, PR should be rebased.

### CODE REVIEW

File: cmd/config/init/interactive.go [hunk 1]
```diff
@@ -21,18 +21,31 @@ func RunInteractiveMode(config *utils.RollappConfig) {
...
+			break
+		}
+		fmt.Println("Expected format: name_uniqueID-revision (e.g. myrollapp_1234-1)")
+	}
 
 	promptDenom := promptui.Prompt{
 		Label:   "Specify your RollApp denom",
 		Default: "RAX",
-		// TODO: add validation for denomination
+		Validate: func(s string) error {
```
[the remark this answers is not in the document]
Comment (mtsitrin) [reply cmd/config/init/interactive.go:40]: It does
It makes validation on the fly, while editing
It gives shitty UX for the rollapp_name, therefore I've implemented it differently for that case

File: cmd/utils/config.go [hunk 2]
```diff
@@ -58,7 +60,14 @@ func (c RollappConfig) Validate() error {
 	}
 	err = ValidateRollAppID(c.RollappID)
 	if err != nil {
-		return fmt.Errorf("invalid RollApp ID '%s'", c.RollappID)
+		return err
```
[the remark this answers is not in the document]
Comment (mtsitrin) [reply cmd/utils/config.go:63]: it's validated in `IsValidDenom`

File: cmd/config/init/interactive.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (ItayLevyOfficial) [reply cmd/config/init/interactive.go:40]: cool cool

**Review by ItayLevyOfficial — APPROVED** [review]
LGTM!

**Pull request merged**